package.json

```diff
@@ -83,7 +84,7 @@
     "postinstall-prepare": "^1.0.1",
     "puppeteer": "^1.1.1",
     "sinon": "^2.3.5",
-    "typescript": "^2.8.0-rc",
+    "typescript": "2.9.0-dev.20180323",
```

Comment:
typescript 2.8.1 just came out with all the typescript features we need, but unfortunately the PR that extends them to javascript didn't get pulled over to the 2.8 branch and now won't be seen until 2.9  (https://github.com/Microsoft/TypeScript/pull/22692#issuecomment-376675329). I did some testing and this is the closest nightly to the 2.8.1 release that has the commit we need and is stable, so I went with this. They're on a 6-8 week cadence, so hopefully we can switch to 2.9 in May-ish